met-web/src/components/engagement/form/RichTextEditor.tsx

```diff
@@ -37,8 +55,8 @@ const RichTextEditor = ({
     };
 
     useEffect(() => {
-        setEditorState(getEditorState(initialRawEditorState));
-    }, [initialRawEditorState]);
+        getStateFromInitialValue(initialRawEditorState, initialHTMLText);
```

Comment:
you don't need to pass them to the function. The function is already in the same block scope of those variables. It can just access them